pages/linux/smbclient.md

```diff
@@ -0,0 +1,19 @@
+# smbclient
+
+> FTP-like client to access SMB/CIFS resources on servers.
+
+- Connect to a share (`exit` to quit the session):
```

Comment:
By default, will it prompt for a username/password ?

Reply:
It will prompt for password, default username will be your current user.

Reply:
You can specify that within the brackets.